src/utilities/__tests__/getIntrospectionQuery-test.js

```diff
@@ -18,7 +18,19 @@ describe('getIntrospectionQuery', () => {
     );
   });
 
-  it('include "isRepeatable" field', () => {
+  it('include "isRepeatable" field on directives', () => {
```

Comment:
Is this related to the isRepeatable change? Or just piggybacking in more tests?

Reply:
@leebyron Thanks for the review 👍 
Just copy-pasted tests for `isRepeatable` as starting point for testing that schema's description is present but forgot to change them.
I wrote a new test and committed them.